### ISSUE TITLE
Blender: Load pointcache loses transformation on top group/empty

### ISSUE THREAD

**Issue by BigRoy** [ynput/ayon-core#529]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior:

This logic [here](https://github.com/BigRoy/ayon-core/blob/8549054098355f20296aca0f994c6a253b820377/client/ayon_core/hosts/blender/plugins/load/load_abc.py#L81-L92) makes it so that the objects are re-parented to an 'asset group' but also make it so that any transformation on the loaded top group is lost.

### Expected Behavior:

Data should not be lost, and preferably any transform or data in the published output is retained on load.

### Version

1.0.0

### What platform you are running on?

Windows

### Steps To Reproduce:

1. Create scene with cube
2. Parent the cube to an empty
3. Move the empty that has the cube as child.
4. Publish the empty as a `pointcacheMain`
5. Load the pointcache

Now the transformation is lost on load.

### Are there any labels you wish to add?

- [X] I have added the relevant labels to the bug report.

**Comment by BigRoy** [comment]
@LiborBatek can you confirm. @simonebarbieri is there any reason to have load work this way?

**Comment by LiborBatek** [comment]
@BigRoy yes, its breaking the transforms if the asset got version update via ```manage``` tool.

There is a way to not scratch the ```alembic``` asset completely when changing ```version``` via changing just the file path leading to the abc file instead within the pointcache modifier. Something which we have discussed it would make much more sense but never happened.

I could give more info or record some vid describing the issue if anyone interested...

**Comment by LiborBatek** [comment]
@BigRoy hmm, reading the issue again Im not sure if this PR refers to loosing any local transform edits on the loaded asset when updated version later on or something else?
